google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta2/CivilTimeEncoderTest.java

```diff
@@ -0,0 +1,362 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigquery.storage.v1beta2;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.logging.Logger;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.threeten.bp.LocalDateTime;
+import org.threeten.bp.LocalTime;
+
+@RunWith(JUnit4.class)
+public class CivilTimeEncoderTest {
+  private static final Logger LOG = Logger.getLogger(CivilTimeEncoderTest.class.getName());
+
+  // Time
+  @Test
+  public void encodePacked64TimeMicros_validTime() {
+    // 00:00:00.000000
+    // 0b000000000000000000000000000|00000|000000|000000|00000000000000000000
+    // 0x0
+    assertEquals(0x0L, CivilTimeEncoder.encodePacked64TimeMicros(LocalTime.of(0, 0, 0, 0)));
```

Comment:
Let's just add a valid decode check after every valid encode case:
assertEquals(LocalTime.of(0, 0, 0, 0), CivilTimeEncoder.decodePacked64TimeMicros(0x0L));

Reply:
I do have those cases in the decodePackd64TimeMicros_validBitFieldTimeMircros() test. Should they be combined into one test?